MdeModulePkg/Library/SecurityLockAuditLibNull/SecurityLockAuditLibNull.inf

```diff
@@ -0,0 +1,26 @@
+##
+#
+#  Null library for security lock logging that does nothing but meet compile requirements
+#
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
```

Comment:
Nit: Can you clean up some of the extra space in these file headers? This PR is a good time for tidying up like this.